tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -236,6 +264,10 @@ def open_browsing_privacy_settings(self):
         self._tap_menu_item(self._browsing_privacy_item_locator)
         return BrowsingPrivacy(self.marionette)
 
+    # method to open menu item that does not return object
+    def simple_open_menu(self, menu_locator):
```

Comment:
Agreed with @mwargers about the naming. Starting a function with an _ is a convention in Python for saying that a function of an attribute is protected.

That said, this won't solve another issue: If we pass the locator itself, we currently pass protected variables (the locators). To solve this we can either:

* Use the functions `open_*_settings()` already defined
* Or make the locators public.

I'm more in favor of the first option. The fact that `open_*_settings()` return another page object is not problematic, we're not obliged to use them.